### PR TITLE
chore: Add mock server instructions

### PR DESCRIPTION
**Pull request by CharlesDuboisSAP**

### CODE REVIEW

File: exercises/01-getting-started.md [hunk 1]
```diff
@@ -43,4 +43,5 @@ npm run watch:local
 
 Now, you should find your application running at http://localhost:8080.
 
+All the data is in September:
```
Comment (marikaner) [inline exercises/01-getting-started.md:46]:
```suggestion
All data is in September:
```

File: exercises/06-use-odata-client.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
 # Create a request with your own OData client
 
+## Deploy the S/4HANA mock server (Optional)
+
+Note: If you have access to an SAP Successfactors Cloud system, you can skip this step.
+
+Download the [SAP SFSF mock server](https://github.wdf.sap.corp/MA/sfsf-mock-server) in a new folder.
```
Comment (marikaner) [inline exercises/06-use-odata-client.md:7]: also not accessible from outside

**Comment by marikaner** [comment]
Any updates?

**Comment by CharlesDuboisSAP** [comment]
@marikaner I think I'll add the instructions about adding your own services to the public mock server

**Comment by marikaner** [comment]
Makes sense. And then just add a link here.

**Comment by CharlesDuboisSAP** [comment]
The [mock server instructions](https://github.com/SAP/cloud-s4-sdk-book/pull/36) (in the readme) are referenced on the getting started page

File: exercises/01-getting-started.md [hunk 3]
```diff
@@ -20,6 +20,18 @@ On your laptop open a command line and enter the following command. Don't forget
 git clone https://github.com/<your-github-username>/cloud-sdk-team-calendar.git
 ```
 
+## Deploy the S/4HANA mock server (Optional)
+
+Note: If you have access to an SAP S/4HANA Cloud system, you can skip this step.
+
+Follow these [instructions](https://github.com/CharlesDuboisSAP/cloud-s4-sdk-book/tree/mock-server#how-to-run-the-server) on how to setup your S4/HANA mock server and deploy it on SAP Cloud Platform, Cloud Foundry.
```
Comment (marikaner) [inline exercises/01-getting-started.md:27]: This should not point to your own github repo, but the one from sap-samples.

File: exercises/06-use-odata-client.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
 # Create a request with your own OData client
 
+## Deploy the S/4HANA mock server (Optional)
```
Comment (marikaner) [inline exercises/06-use-odata-client.md:3]: Move it up to the other section.

**Comment by marikaner** [comment]
Ah one more thing, in the `.env` file maybe you can replace the urls with `url: '<s4-server-url>'` (and sfsf of course)

**Pull request merged**